package.json

```diff
@@ -98,7 +96,9 @@
     "single-spa": "^5.9.3",
     "single-spa-react": "^4.4.0",
     "style-loader": "^3.3.1",
```

Comment:
style-loader can be a dev dep
@types/react-router-dom can be a dev dep

idk what qs is used for, I don't see any usage. but we are uncertain, let me be for now

Reply:
thank you, I changed it